reactivesocket-cpp/src/ReactiveStreamsCompat.h

```diff
@@ -2,7 +2,7 @@
 
 #pragma once
 
-#include "reactive-streams-cpp/ReactiveStreams.h"
+#include <ReactiveStreams.h>
```

Comment:
Nope. Conflict with another library that has a header called ReactiveStreams.h _anywhere_ on the include path (dozens locations, at best). Those are rather subtle bugs where you think the compiler picked up different header than the one you think it did. Obviously, if those two headers differed between parts of a build process (e.g. because one file included stuff using "" syntax and the other with <> syntax), everything could be fine until you run the binary... To make matters worse, you can't depend on the order of folders on include path, as it's not defined in the standard http://en.cppreference.com/w/cpp/preprocessor/include.

Look at how other libraries install headers -- usually (always?) in a subdirectory named after that library: folly, event2, wangle, boost, etc. Whatever you put in /usr/local/include/ will be picked up by all C++ builds on that machine...